server/models/Group.js

```diff
@@ -448,6 +449,15 @@ export default function(Sequelize, DataTypes) {
     },
 
     classMethods: {
+      createMany: (groups, defaultValues) => {
+        return Promise.map(groups, group => {
+          for (const attr in defaultValues) {
```

Comment:
I had the impression that for/while loops are generally discouraged, in favor of maps, forEach, etc. I don't know where I got that from though.